### DISCUSSION TITLE
Access a single value for a row column pair by integer position

### DISCUSSION THREAD

**Discussion by gregory-chatelier** [opensource9ja/danfojs#219] [Ideas]
Hi everyone, 
The documentation lacks examples on how to access a single value by integer position on both axes.

"Pandas" has a convenient "iat" method that allows direct assignation : 
df.iat[1, 2] = 10

I assumed that with "danfojs" the same result could be accomplished with :
df.values
and for assignation :
df.values[1][2] = 10

It seems to work, but I'm not 100% sure if it is the right an efficient way to do that

**Comment by risenW** [comment]
@gregory-chatelier This feature has not been implemented yet. Using `df.values[1][2] = 10` does not work properly as the internal Data structure is not updated, just the data being displayed. 

A correct solution when implemented, will update the Data Frame when you update a cell.

**Reply by gregory-chatelier** [reply]
I now have a better overall understanding of the library, I see it's using immutable Tensors and that by design very few operations are applicable in place.

For instance there seems to be no way a Serie can be replaced without redefining the Dataframe. One can drop a Serie, or add a column but the indices are not preserved, a Serie does not replace a previous one.

This is in fact counterintuitive, I would have expected more mutability for performance reasons.

Now, I'm asking myself if this really worth rebuilding a 100K+ rows Dataframe each time a value needs some sort of dynamic input.

**Reply by risenW** [reply]
@gregory-chatelier we now support in place operation for all mutable operations . Try out v0.3.0
